saleor/product/tasks.py

```diff
@@ -5,6 +5,7 @@
 
 
 def _update_variants_names(instance, saved_attributes):
+    """update saved variants name for products type."""
```

Comment:
I'd like to see an explanation, of how the variant's name is composed as that's not obvious

Reply:
Also, we should start with capital letter